persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalFileConstants.java

```diff
@@ -230,8 +231,17 @@ private InheritedTypeReferenceConstants() {
     public static final String GET_MANY_RELATIONS = "check " +
             "(<persist:SQLClient>self.persistClient).getManyRelations(nextRecord.value," +
             " <%sRelations[]>self.include);";
-    public static final String PLACEHOLDER_PATTERN = "\"\\$\\{(.*?)}\"";
-    public static final String MYSQL_DRIVER_IMPORT = "import ballerinax/mysql.driver as _;";
+
+    public static final String COMMENT_AUTOGENERATED = "// AUTO-GENERATED FILE. DO NOT MODIFY.";
+
+    public static final String COMMENT_GENERATED_BY_BALLERINA =
+            "// This file is an auto-generated file by Ballerina persistence layer.";
+
+    public static final String COMMENT_GENERATED_BY_BALLERINA_PERSIST =
```

Comment:
```suggestion
    public static final String AUTO_GENERATED_COMMENT_WITH_REASON =
```

Reply:
Updated in [f524212](https://github.com/ballerina-platform/persist-tools/pull/79/commits/f5242122d14dbc8c86354caaecf145c00733a2f7)